examples/lock-app/cc13x2x7_26x2x7/main/AppTask.cpp

```diff
@@ -151,6 +151,22 @@ int AppTask::Init()
             ;
     }
 
+#if CHIP_ENABLE_ADDITIONAL_DATA_ADVERTISING && CHIP_ENABLE_ROTATING_DEVICE_ID
+    {
+	uint8_t EUI64[8];
+        char stringEUI64[sizeof(EUI64) * 2 + 1]; // just big enough for string(EUI64)
+
+        // set serial number to flash
+        // this is before ZCL is started, so Basic Cluster populates Serial Number with this EUI64 (in string)
+        ThreadStackMgrImpl().GetIeeeEui64(EUI64);
+        chip::Encoding::BytesToUppercaseHexString(EUI64, sizeof(EUI64), stringEUI64, sizeof(stringEUI64));
+	ret = ConfigurationMgr().StoreSerialNumber(stringEUI64, strlen(stringEUI64));
+	PLAT_LOG("StoreSerialNumber: %s", stringEUI64);
+	VerifyOrDieWithMsg(ret == CHIP_NO_ERROR, NotSpecified,
+			   "StoreSerialNumber: failed (%lX)", ret.AsInteger());
```

Comment:
```suggestion
			   "StoreSerialNumber: failed (%" CHIP_ERROR_FORMAT ")", ret.Format());
```
so this will work right if verbose error formatting is enabled.

Reply:
Tested it out, and it works, thanks!
Updated here: https://github.com/project-chip/connectedhomeip/pull/15367/commits/ae01057e16c3ce4e52e70d08b5f24f6784febecb